src/bench/bench_bitcoin.cpp

```diff
@@ -137,14 +132,14 @@ int main(int argc, char** argv)
 
     try {
         benchmark::Args args;
-        args.asymptote = parseAsymptote(argsman.GetArg("-asymptote", ""));
-        args.is_list_only = argsman.GetBoolArg("-list", false);
-        args.min_time = std::chrono::milliseconds(argsman.GetIntArg("-min-time", DEFAULT_MIN_TIME_MS));
-        args.output_csv = argsman.GetPathArg("-output-csv");
-        args.output_json = argsman.GetPathArg("-output-json");
-        args.regex_filter = argsman.GetArg("-filter", DEFAULT_BENCH_FILTER);
-        args.sanity_check = argsman.GetBoolArg("-sanity-check", false);
-        args.priority = parsePriorityLevel(argsman.GetArg("-priority-level", DEFAULT_PRIORITY));
+        args.asymptote = parseAsymptote(AsymptoteSetting::Get(argsman));
```

Comment:
I understand that often search and replace can change a lot of code and is very convenient for the author, but all those unrelated changes are very hard to review.
Would it be possible to go end-to-end for a very narrow part of the settings and slowly strangle out the old version (i.e. depth-first-search instead of breadth first search)?